nntrainer/tensor/manager.cpp

```diff
@@ -369,6 +368,10 @@ std::vector<Weight *> Manager::requestWeights(
     auto &[dim, t_initializer, w_reg, w_reg_const, decay, clip_by_global_norm,
            need_gradient, name] = weights_spec.at(i);
     auto grad_exec_order = default_grad_exec_order;
+
+    if (trainable)
```

Comment:
This is trivial but I think grad_exec_order already used only the layer is trainable so this change will not affect the weights lifespan.

Reply:
Yeah you're right.
I was confused.